pkg/action/discoverer.go

```diff
@@ -173,6 +174,7 @@ func (d *Discoverer) getTargets(ctx context.Context) ([]*targetgroup.Group, erro
 						"project", project,
 						"vapp", vappName,
 						"server", server.Name,
+						"ID", server.ID,
```

Comment:
The log labels are all lowercase, so let's keep that :)

Reply:
applied in [8bb507d](https://github.com/promhippie/prometheus-vcd-sd/pull/31/commits/8bb507df716907a7279d67e92d7664cf918b5fd4)